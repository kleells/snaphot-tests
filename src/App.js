import React from 'react'
import './App.css'
import { useState, useEffect } from 'react';
import GitHubCard from './GitHubComponents/GitHubCard';
import Row from 'react-bootstrap/Col';
import Col from 'react-bootstrap/Row';

function App() {
  return (
    <div className="App pt-5">
      <Row>
        <Col style={{display: 'flex', justifyContent: 'center'}}>
          <GitHubCard/>
        </Col>
      </Row>
    </div>
  );
}

export default App;
