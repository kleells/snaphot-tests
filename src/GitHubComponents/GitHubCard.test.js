import GitHubCard from './GitHubCard'
import renderer from 'react-test-renderer'
import App from '../App'

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard />).toJSON()
  expect(tree).toMatchSnapshot()
})