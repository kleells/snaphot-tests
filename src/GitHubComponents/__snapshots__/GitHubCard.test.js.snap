// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "border": "none",
    }
  }
>
  <div
    className="card"
    style={
      Object {
        "margin": "auto",
        "width": "18rem",
      }
    }
  >
    <img
      alt="Github profile image"
      className="card-img-top"
      height="200px"
      src=""
      width="200px"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      />
      <p
        className="card-text"
      />
      <p
        className="card-text"
      >
        <a
          href=""
        >
          <button
            className="btn btn-primary"
            disabled={false}
            type="button"
          >
            Link to GitHub
          </button>
        </a>
      </p>
    </div>
  </div>
</div>
`;
